gulpfile.js/lib/isProductionBuild.js

```diff
@@ -0,0 +1,5 @@
+const argv = require('minimist')(process.argv.slice(2));
+
+module.exports = () => {
```

Comment:
Lijkt me nutteloos om voor dit telkens opnieuw een function call te doen niet?
Al moet je het ook niet doen voor de perfomance winst

Reply:
Hmm yeah, maybe... Can't really find a good argument to do this except: 'it feels better to me' 😅